modules/renter/upload.go

```diff
@@ -140,10 +141,11 @@ func (r *Renter) Upload(up modules.FileUploadParams) error {
 	// number of pieces plus some buffer before we decide that an upload is
 	// okay.
 	if len(r.hostDB.ActiveHosts()) < 1 {
-		return errors.New("not enough hosts on the network to upload a file :( - maybe you need to upgrade your software")
+		return errors.New("not enough hosts on the network to upload a file")
```

Comment:
oh or they are having network problems or something. I guess there are more troubleshooting things than just the one statement.